templates/thumbor.conf.j2

```diff
@@ -1,45 +1,130 @@
-# Thumbor imaging service config file (template created with Tumbor 6.4.1)
+# Thumbor imaging service config file (template created with Tumbor 6.7.0)
 # See https://github.com/thumbor/thumbor/wiki/Configuration
 # {{ ansible_managed }}
 
+################################### Logging ####################################
+
+## Logging configuration as json
+## Defaults to: None
+#THUMBOR_LOG_CONFIG = None
+
+## Log Format to be used by thumbor when writing log messages.
+## Defaults to: '%(asctime)s %(name)s:%(levelname)s %(message)s'
+#THUMBOR_LOG_FORMAT = '%(asctime)s %(name)s:%(levelname)s %(message)s'
+
+## Date Format to be used by thumbor when writing log messages.
+## Defaults to: '%Y-%m-%d %H:%M:%S'
+#THUMBOR_LOG_DATE_FORMAT = '%Y-%m-%d %H:%M:%S'
+
+################################################################################
+
 
 ################################### Imaging ####################################
 
 ## Max width in pixels for images read or generated by thumbor
-MAX_WIDTH = 5000
+## Defaults to: 0
+#MAX_WIDTH = 0
 
 ## Max height in pixels for images read or generated by thumbor
-MAX_HEIGHT = 5000
+## Defaults to: 0
+#MAX_HEIGHT = 0
 
 ## Max pixel count for images read by thumbor
-MAX_PIXELS = 75000000
+## Defaults to: 75000000.0
+#MAX_PIXELS = 75000000.0
 
 ## Min width in pixels for images read or generated by thumbor
-MIN_WIDTH = 2
+## Defaults to: 1
+#MIN_WIDTH = 1
 
 ## Min width in pixels for images read or generated by thumbor
-MIN_HEIGHT = 2
+## Defaults to: 1
+#MIN_HEIGHT = 1
 
 ## Allowed domains for the http loader to download. These are regular
 ## expressions.
+## Defaults to: [
+#]
+{# % if thumbor_allowed_sources is defined and thumbor_allowed_sources is not none %#}
```

Comment:
Is there a syntax problem here?

Reply:
Nice catch. I had forgotten to remove this after I was done testing. Let me remove it.